Code/Editor/AzAssetBrowser/AzAssetBrowserWindow.cpp

```diff
@@ -624,15 +632,17 @@ void AzAssetBrowserWindow::UpdateBreadcrumbs(const AzToolsFramework::AssetBrowse
     using namespace AzToolsFramework::AssetBrowser;
 
     QString entryPath;
+    QString fullPath;
     if (selectedEntry)
     {
         const AssetBrowserEntry* folderEntry = Utils::FolderForEntry(selectedEntry);
         if (folderEntry)
         {
-            entryPath = QString::fromUtf8(folderEntry->GetVisiblePath().c_str());
+            entryPath = FromStdString(folderEntry->GetVisiblePath().c_str());
+            fullPath = FromStdString(folderEntry->GetFullPath().c_str());
```

Comment:
Don't pass the result of `.c_str()` to a `string_view` parameter, it degrades the type down to a `const char*` and loses the length information.
```suggestion
            entryPath = FromStdString(folderEntry->GetVisiblePath());
            fullPath = FromStdString(folderEntry->GetFullPath());
```

Reply:
Fixed!